adam/language_specific/english/english_phase_1_lexicon.py

```diff
@@ -127,6 +130,9 @@
         (HEAD, LexiconEntry("head", NOUN, plural_form="heads")),
         (PAPER, LexiconEntry("paper", NOUN, plural_form="papers")),
         (BEAR, LexiconEntry("bear", NOUN, plural_form="bears")),
+        (COW, LexiconEntry("cow", NOUN, plural_form="cows")),
+        (BEEF, LexiconEntry("beef", NOUN, plural_form="beefs")),
+        (CHICKEN, LexiconEntry("chicken", NOUN, plural_form="chickens")),
```

Comment:
These need to be added to the Chinese lexicon or we could hit issue generating curriculum in Chinese.